elixir/config/config.exs

```diff
@@ -75,6 +75,17 @@ config :domain, Domain.GoogleCloudPlatform,
   sign_endpoint_url: "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/",
   cloud_storage_url: "https://storage.googleapis.com"
 
+config :domain, Domain.ComponentVersions,
+  firezone_releases_url: "https://www.firezone.dev/api/releases",
+  from_url: true,
+  versions: [
+    apple: "1.0.0",
```

Comment:
I would put here the last deployed version that we do have now. This will allow us to keep at least some notifications even if website API is down after a node is rebooted

Reply:
👍 updated